doc/man7/provider-keymgmt.pod

```diff
@@ -329,13 +333,21 @@ OSSL_FUNC_keymgmt_export() should extract values indicated by I<selection>
 from I<keydata>, create an L<OSSL_PARAM(3)> array with them and call
 I<param_cb> with that array as well as the given I<cbarg>.
 
-OSSL_FUNC_keymgmt_import_types() should return a constant array of descriptor
+OSSL_FUNC_keymgmt_import_types() and OSSL_FUNC_keymgmt_import_types2()
+should return a constant array of descriptor
 L<OSSL_PARAM(3)> for data indicated by I<selection>, for parameters that
 OSSL_FUNC_keymgmt_import() can handle.
+Either OSSL_FUNC_keymgmt_import_types() or OSSL_FUNC_keymgmt_import_types2(),
+must be implemented, if OSSL_FUNC_keymgmt_import_types2() is implemented, then
+it is preferred over OSSL_FUNC_keymgmt_import_types().
 
-OSSL_FUNC_keymgmt_export_types() should return a constant array of descriptor
+OSSL_FUNC_keymgmt_export_types() and OSSL_FUNC_keymgmt_export_types2()
+should return a constant array of descriptor
 L<OSSL_PARAM(3)> for data indicated by I<selection>, that the
 OSSL_FUNC_keymgmt_export() callback can expect to receive.
+Either OSSL_FUNC_keymgmt_export_types() or OSSL_FUNC_keymgmt_export_types2(),
+must be implemented, if OSSL_FUNC_keymgmt_export_types2() is implemented, then
+it is preferred over OSSL_FUNC_keymgmt_export_types().
```

Comment:
Should we note somewhere that you should implement the original import_types/export_types functions if your provider is seeking backwards compatibility with 3.0?

Reply:
Changed to code to use _ex notation. 

Also added note in provider-keymgmt.pod about OpenSSL 3.0 (and 3.1) compatibility.
I assume that this does not go into 3.1, so we need to mention backward compatibility to 3.0 and 3.1, right?

Reply:
Correct